runtime/default.go

```diff
@@ -204,19 +78,53 @@ func (r *runtime) run() {
 			if service.Running() {
 				continue
 			}
-
 			// TODO: check service error
-			log.Debugf("Starting %s", service.Name)
+			log.Debugf("Runtime starting service %s", service.Name)
 			if err := service.Start(); err != nil {
-				log.Debugf("Runtime error starting %s: %v", service.Name, err)
+				log.Debugf("Runtime error starting service %s: %v", service.Name, err)
+			}
+		case event := <-events:
+			log.Debugf("Runtime received notification event: %v", event)
+			// NOTE: we only handle Update events for now
+			switch event.Type {
+			case Update:
+				// parse returned response to timestamp
+				buildTime, err := time.Parse(time.RFC3339, event.Version)
+				if err != nil {
+					log.Debugf("Runtime error parsing build time: %v", err)
+					continue
+				}
+				r.Lock()
+				for _, service := range r.services {
```

Comment:
Event now has a Service string, so we should probably only update the specific service the event is for. Unless the Service name is blank in which case I guess you could assume a blanket update across all things.